<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuAction</class>
 <widget class="QWidget" name="MenuAction">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>147</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Menu Action</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color:#2A2E3A;
    border:none;
}

QWidget#MenuAction:hover {
    background-color:#414755;
    border:none;
}

QLabel {
    border:none;
    padding: 6 2 6 11;
    color: #F2F3F3;
    font: normal 9pt &quot;Tahoma&quot;;
}

#cbxSelected {
    color: #F2F3F3;
    font: normal  9pt &quot;Tahoma&quot;;
}

#cbxSelected::indicator {
    border:none;
    width: 23px;
    height: 30px;
}

#cbxSelected::indicator:checked {
    subcontrol-position: left;
    padding-left: 6;
    image: url(:/img/tick_menu.png);
    image-position: left;
}

#cbxSelected[hovered=&quot;true&quot;] {
    background-color:#414755;
}

#lblName[hovered=&quot;true&quot;] {
    background-color:#414755;
}

#lblValue[hovered=&quot;true&quot;] {
    background-color:#414755;
}

#pbnIcon {
    padding: 2 8 0 0;
    image: url(:/img/arrow_right.png);
}

#pbnIcon:hover {
    background-color:#414755;
    image: url(:/img/arrow_right_hover.png);
}

#pbnIcon:pressed {
    image: url(:/img/arrow_right_pressed.png);
}

#pbnIcon[hovered=&quot;true&quot;] {
    background-color:#414755;
    image: url(:/img/arrow_right_hover.png);
}

#pbnIcon[state=empty] {
    image: url(:/img/arrow_right_empty.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="cbxSelected">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Value </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbnIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
